CHANGELOG.md

```diff
@@ -1,5 +1,152 @@
 # CHANGELOG
 
+## v1.0.0
+*August 22nd, 2022*
+
+After more than 2 years in the works, this is the first stable release of the Hermes relayer! 🎉
```

Comment:
For reaching this milestone, we thank the valuable contributions of over 50 individuals, spread across more than 800 documented & resolved issues. Beside Cosmos-SDK and Tendermint, we are fortunate to maintain some of the most active and intense repository in the Cosmos ecosystem. Most importantly, we thank the relentless work of relayer operators that have already relayed billions worth of value in IBC production networks, and have provided us with invaluable feedback on improving Hermes and raising the overall stability of IBC. Kudos to everyone!